src/hw_5/t06_find_max_divisor.py

```diff
@@ -0,0 +1,16 @@
+""" Find max divisor
+
+Find a maximum divisor d of a number n, d = 2**k.
+Example:
+    10(2) 16(16), 12(4).
+"""
+
+
+def get_max_divisor(n):
```

Comment:
amazing. 
I don't understand how have you found this solution. 
My solution was with shifts till last wouldn't be 1.

Reply:
Your solution was my first idea. Then I spent one day solving tasks with bit operators. I saw the  idea to use n & (n-1) in another task and puzzle was done! I also google one more solution.  
x = n& -n !!!